src/main/rules/GCI604/java/GCI604.asciidoc

```diff
@@ -0,0 +1,128 @@
+When two classes are in One To Many relation and LAZY fetching strategy, calling the child list in a loop can provoque unnecessary SQL queries. This will both hinder the application performance and increase environmental impact.
```

Comment:
Could you please add some documentation to prove this rule ? (some web documentation or local tests for example)

Reply:
like this one : https://www.baeldung.com/spring-hibernate-n1-problem